cernopendata/modules/fixtures/data/records/lhcb-antimatter-matters-2017.json

```diff
@@ -0,0 +1,196 @@
+  ], 
+  "methodology": {
+    "description": "Offline Selections cuts were made on: track transverse momentum (pT), sum of pT of Tracks, Track momentum, B candidate mass assuming all tracks are Kaons, Track impact parameter chi-square, sum of impact parameter chi-square of tracks, B candidate vertex fit chi-square."
+  },
+  "publisher": "CERN Open Data Portal",
+  "recid": "FIXME",
+  "title": "Matter Antimatter Differences (B meson decays to three hadrons) - Data Files",
+  "type": {
+    "primary": "Dataset",
+    "secondary": []
+  },
+  "usage": {
+    "description": "The data files can be accessed using ROOT.",
+    "links": [
+      {
+        "url": "https://root.cern.ch/"
+      }
+    ]
+  },
+  "use_with": {
+    "description": "The data files are expected to be used with the Project Notebook.",
+    "links": [
+      {
+        "recid": "FIXME_Software-record"
+      }
+    ]
+  },
+  "validation": {
+    "description": "The LHCb collaboration performs online and offline monitoring of the quality of the data collected. Only data for which the primary detector subsystems are highly performant are used for analysis.",
+    "links": [
+      {
+        "description": "The performance of the detector is described in", 
+        "url": "https://cds.cern.ch/record/1978280?ln=en"
+      }
+    ]
+  }
+}
+,
+{
+  "abstract": {
+    "description": "This documentation provides a basic description of the contents of the project and instructions on how to get started."
+  },
+  "accelerator": "CERN-LHC",
+  "collaboration": {
+    "name": "LHCb collaboration"
+  },
+  "collections": [
+    "LHCb-Learning-Resources"
+  ],
+  "date_published": "2017",
+  "distribution": {
+    "formats": [
+      "pdf"
+    ], 
+    "number_files": 1,
+    "size": 496943
+  },
+  "doi": "10.7483/OPENDATA.LHCB.IID3.87CV",
+  "experiment": "LHCb",
+  "files": [
+    {
+      "size": 496943, 
+      "uri": "FIXME_Upload-Notebook-Instructions"
+    }
+  ], 
+  "publisher": "CERN Open Data Portal",
+  "recid": "FIXME",
+  "title": "Matter Antimatter Differences (B meson decays to three hadrons) - Introductory Documentation",
+  "type": {
+    "primary": "Documentation",
+    "secondary": []
+  },
+  "use_with": {
+    "description": "The data files and the project notebook are available at",
+    "links": [
+      {
+        "recid": "FIXME_Data-record"
+      },
+      {
+        "recid": "FIXME_Software-record"
+      }
+    ]
+  }
+}
+,
+{
+  "abstract": {
+    "description": "This project notebook provides a guided analysis of data from the LHCb experiment. The analysis searches for a difference between the behavior of matter and anti-matter. The project analyses B-meson decays to three charged particles."
+  },
+  "accelerator": "CERN-LHC",
+  "collaboration": {
+    "name": "LHCb collaboration"
+  },
+  "collections": [
+    "LHCb-Tools"
+  ],
+  "date_published": "2017",
+  "distribution": {
+    "formats": [
+      "FIXME"
+    ], 
+    "number_files": 34,
```

Comment:
We only store one tarball file, so number of files is 1 and the size is 5204130.